music_assistant/server/providers/deezer/__init__.py

```diff
@@ -71,6 +72,22 @@
     ProviderFeature.RECOMMENDATIONS,
 )
 
+
+@dataclass
+class Credential:
+    """Class for storing credentials."""
+
+    def __init__(self, app_id: int, app_secret: str, access_token: str):
+        """Set the correct things."""
+        self.app_id = app_id
+        self.app_secret = app_secret
+        self.access_token = access_token
```

Comment:
this is redundant in case of a dataclass

Reply:
Oh yeah..